modules/localgov_directories_page/tests/src/Functional/PathIntegrationTest.php

```diff
@@ -22,7 +22,7 @@ class PathIntegrationTest extends BrowserTestBase {
    *
    * @var string
    */
-  protected $profile = 'standard';
+  protected $profile = 'testing';
```

Comment:
To nit-pick can this not just get deleted? It's only needed if it's overriding the value.